### PR TITLE
Setup label command.

### PR DESCRIPTION
**Pull request by diepsohung**

### CODE REVIEW

File: lib/3llo/api/label.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      end
+
+      def find(label_id)
+        req_path = Utils.build_req_path("/labels/#{label_id}")
+        label_payload = client.get(req_path, {})
+
+        make_struct(label_payload)
+      end
+
+      def create(name, color, board_id)
```
Comment (qcam) [inline lib/3llo/api/label.rb]:
```suggestion
      def create(name:, color:, board_id:)
```

File: lib/3llo/utils.rb [hunk 2]
```diff
@@ -17,6 +17,8 @@ module Utils
       "sky" => 36
     }.freeze()
 
+    TRELLO_LABEL_COLOR = %w[red pink blue green purple yellow orange sky].freeze()
```
Comment (qcam) [inline lib/3llo/utils.rb]: You can use `Hash#keys`

Comment (diepsohung) [reply lib/3llo/utils.rb]: There are some unnecessary keys Trello doesn't support as cyan and lime. It's not really friendly when you let user choose and return the unexpected response.

Comment (qcam) [reply lib/3llo/utils.rb]: ok that makes sense.

File: lib/3llo/entities.rb [hunk 3]
```diff
@@ -11,7 +11,7 @@ module Entities
     Board = Struct.new(:id, :shortcut, :name)
     List = Struct.new(:id, :shortcut, :name)
     Card = Struct.new(:id, :shortcut, :name, :description, :short_url, :labels, :members, :list, keyword_init: true)
-    Label = Struct.new(:name, :color)
+    Label = Struct.new(:id, :shortcut, :name, :color)
```
Comment (qcam) [inline lib/3llo/entities.rb]:
```suggestion
    Label = Struct.new(:id, :shortcut, :name, :color, keyword_init: true)
```

I think let's use keyword init.

Comment (diepsohung) [reply lib/3llo/entities.rb]: Added, please check it again.

**Pull request merged**